client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -91,19 +112,29 @@ class Unlocked extends React.Component {
 
 	render() {
 		const { translate } = this.props;
-		const { privateDomain, hasPrivacyProtection, manualTransferRequired, pendingTransfer } = getSelectedDomain( this.props );
-		let domainStateMessage = translate( 'Your domain is unlocked to prepare for transfer.' );
+		const {
+			privateDomain,
+			hasPrivacyProtection,
+			manualTransferRequired,
+			pendingTransfer,
+			domainLockingAvailable,
+		} = getSelectedDomain( this.props );
+
+		let domainStateMessage;
 
 		if ( pendingTransfer ) {
 			domainStateMessage = translate( 'Your domain is pending transfer.' );
-		} else if ( hasPrivacyProtection && ! privateDomain ) {
+		} else if ( domainLockingAvailable && hasPrivacyProtection && ! privateDomain ) {
 			domainStateMessage = translate( 'Your domain is unlocked and Privacy Protection has been disabled' +
 				' to prepare for transfer.' );
+		} else if ( domainLockingAvailable ) {
+			domainStateMessage = translate( 'Your domain is unlocked to prepare for transfer.' );
 		}
+		// Otherwise, we can skip this paragraph.
```

Comment:
What about if we go even more explicit, like `If the domain doesn't support locking don't even mention it, otherwise we'll just confuse the user`?